src/Goerli-DssSpell.t.sol

```diff
@@ -2633,13 +2696,14 @@ contract DssSpellTest is DSTest, DSMath {
         // ----------------------- Check Clipper works and bids can be made -----------------------
 
         {
-        DSTokenAbstract token = DSTokenAbstract(join.gem());
+        BrokeTokenAbstract token = BrokeTokenAbstract(join.gem());
         uint256 tknAmt =  ilkAmt / 10 ** (18 - join.dec());
-        giveTokens(token, tknAmt);
+        giveBrokeTokens(token, tknAmt);
```

Comment:
Maybe simpler to use `giveTokens(DSTokenAbstract(address(token)), tknAmt)` here so as to avoid having to define `giveBrokeTokens`?

Reply:
yeah, was thinking that as well but like:
`giveTokens(BrokeTokenAbstract(address(token)), tknAmt)`

Reply:
yeah, I probably should have.